### DISCUSSION TITLE
Automattic aquisition

### DISCUSSION THREAD

**Discussion by eamonburns** [Automattic/harper#284] [General]
[Link to article](https://wptavern.com/automattic-acquires-harper-founder-elijah-potter-joins-the-team-as-code-wrangler)

I was on the admin page of my WordPress site, and randomly saw a link to this!

This is pretty exciting.

**Comment by elijah-potter** [comment]
Yeah, I'm thrilled. We'll have a lot more resources to make Harper the most capable, private option out there.
